wagtailio/project_styleguide/templates/patterns/components/sign_up_form/base_form.html

```diff
@@ -0,0 +1,17 @@
+<div class="sign-up-form {% if classes %}{{ classes }}{% endif %}">
+    <h2 class="sign-up-form__heading">{{ heading }}</h2>
+    <p class="sign-up-form__sub-heading intro-small">{{ sub_heading }}</p>
+    <form class="sign-up-form__container" action="https://torchbox.us1.list-manage.com/subscribe/post?u={{ mailchimp_account_id }}&id={{ mailchimp_newsletter_id }}" method="post">
+        <fieldset>
+            <label for="email" class="sign-up-form__label"></label>
```

Comment:
**Suggestion:** If this is meant to be hidden then populate the label contents and add the screen reader only class:

`<label for="email" class="u-sr-only sign-up-form__label">Enter your email address</label>`

This can then be deleted in `_sign-up-form.scss`:

```
&__label {
    display: none;
}
 ```